.github/workflows/builds.yml

```diff
@@ -69,6 +69,11 @@ jobs:
                 zlib1g \
                 zlib1g-dev
       - run: cargo install --force --debug cbindgen
+      - run: git clone --depth 1 https://github.com/catenacyber/jsonschema-rs-iter.git
+      - name: Build jsonschema-rs-iter
+        working-directory: jsonschema-rs-iter
+        run: |
+          cargo install --path .
```

Comment:
You could `cargo install --git https://github.com/catenacyber/jsonschema-rs-iter`.

Reply:
Ok, but moved it to S-V repo